arangod/GeneralServer/RequestLane.h

```diff
@@ -160,9 +177,11 @@ inline RequestPriority PriorityRequestLane(RequestLane lane) {
       return RequestPriority::HIGH;
     case RequestLane::CONTINUATION:
       return RequestPriority::MED;
-    case RequestLane::UNDEFINED: {
-      TRI_ASSERT(false);
-    }
+    case RequestLane::UNDEFINED:
+      // assume low priority for UNDEFINED. we should never get
+      // here under normal circumstances. if we do, returning the
+      // default shouldn't do any harm.
+      return RequestPriority::LOW;
```

Comment:
Since we are not supposed to get here, perhaps leave the assert in?

Reply:
I think I can't, because `TRI_ASSERT()` isn't constexpr.